doc: 3.0.0-preview1 doc updates (#20615)
nrfconnect/sdk-nrf: doc/nrf/index.rst

Introduction
############

.. important::

>    |NCS| v3.0.0-preview1 is a development tag used for development purposes but is not recommended to use in a production environment.

Reviewer: ```suggestion
   |NCS| v3.0.0-preview1 is a development tag used only for development purposes.
```
Would this be sufficient?
